README.md

```diff
@@ -1,25 +1,71 @@
-<img src="./media/cta_banner.png"/>
+<a href="https://canopas.com/contact" target="_blank"><img src="./media/cta_banner.png"/></a>
 
 # Bite Space – Your Ultimate Culinary Companion!
 
-Bite Space is a web application designed to help users discover new restaurants and cuisines, explore menus, and find their next favorite dish. Whether you're a foodie looking for your next culinary adventure or a restaurant owner seeking to expand your reach, FoodFinder has you covered.
+Bite Space is a web application designed to help users discover new restaurants and cuisines, explore menus, and find their next favorite dish. Whether you're a foodie looking for your next culinary adventure or a restaurant owner seeking to expand your reach, Bite Space has you covered.
```

Comment:
Bite Space is a web application designed to help users discover new restaurants and cuisines, explore menus, and find their next favorite dish.

to 

Bite space is a platform, where users can discover new restaurants and cuisines, explore menus, and find their next favorite dish.